lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss

```diff
@@ -24,56 +28,59 @@
 
 
 
-// Main Reportback container
 .reportback {
-  > .wrapper {
-    @include clearfix();
-    position: relative;
-    width: 100%;
+  padding-bottom: $base-spacing;
 
-    @include media($tablet) {
-      display: table;
+  > .wrapper {
+    @include media($medium) {
       direction: rtl;
+      display: table;
       table-layout: fixed;
+      width: 100%;
     }
   }
 
-  .view-more {
-    @include clearfix();
+  .spacer {
```

Comment:
like `reportback__spacer`? The styles are namespaced to the `.reportback` module...